doc/source/report/path.rst

```diff
@@ -75,81 +57,78 @@ As we begin our analysis, we would like to explore the following points:
-define an “exploration score” which takes a high value if a mouse goes
-every part of the cage before going back to the nest. - Functions
-possibly needed in this step: - Cut\_paths: cuts out paths into several
-smaller paths
-
-**Step 2**: Choose Key Features
-
-The examples of the key features would include the following: - The
-stain of the mouse - The time of day - Some pre-defined “score” of a
-path e.g. efficiency - How frequently a mouse visits/stays at a location
-- How long it stays at a specific location (ex: nest or food) - The last
-time the mouse ate - The last time the mouse drank water - The total
-distance on average a mouse travels per day - How fast on average a
-mouse travels
-
-We need to construct the functions to generate those features.
-
-**Step 3**: Classification: Machine Learning Technique
-
-We might be able to use several machine learning methods, including
-random forest, SVM, gradient boosting. Each method should have following
-steps: - Cross validation: divide the data into train, validation, and
-test set - Tune the parameters: Based on the train and validation set,
-tune the parameter to maximize some pre-defined measurement. - Fit on
-the test set: Evaluate the performance of the classification on the test
-set.
-
-**Step 4**: Interpretation
-
-Hopefully we might employ the different model like logistic regression
-to get a sense of the effect size of each features on the response
-variables.
+We define a path based on a specific time threshold between movements. In more
+detail, we created a function that considers that time difference. If the time
+between movements exceeded the threshold, we considered a new path to have
+begun. That path was considered to have ended once the threshold had been
+exceeded again. Based on the advice of the Tecott lab, the default threshold
+was one second.
+
+**Step 2**: Clean Data
+
+Based on our definition of a path, we realized that we needed to clean the
+data to remove obvious outliers and noise. For instance, we found that the
+sensor platform generated extreme outliers such as an acceleration of -4000
+centimeters per second. In order to remove this noise, we created two
+functions: one to filter the paths and one to remove duplicate rows. The
+filter paths function uses a minimum number of points in a path to filter out
+paths that create noise. For example, a path that is only three points could
+simply be a mouse shifting weight from one foot to the other and back again.
+Since this is not the type of path we are interested in analyzing, it makes
+sense to filter them out. The remove duplicates function deals with an issue
+in the data where the same x and y coordinates appear in adjacent rows but
+with different timestamps. This caused a problem when computing the angles of
+the paths so, while it is a trivial removal, it is necessary.
+
+**Step 3**: Choose Key Features
+
+The path features we wrapped up into functions are as follows: 
```

Comment:
This line was italicized for some reason. Maybe you want to check rst file style?